lib/getConfig.js

```diff
@@ -7,7 +7,7 @@ const getConfig = (altPath) => {
   const pathString = altPath || path.join(cwd, '.sgcrc');
   let configObject = json.readToObjSync(pathString);
 
-  if (!configObject) {
+  if (configObject) {
```

Comment:
Yes that is true, but there is another problem. If the user has no `sgc` specified in the `package.json`, it will throw an error, and will never get into the second if stmt. So it will never take the default types from `.sgcrc_default`